ReleaseNotes.md

```diff
@@ -1,5 +1,56 @@
 # Release Notes
 
+## New in Release 2.5.0
+Compression-aware training:
+- New Features:
+  - Added `nncf.common.utils.patcher.Patcher` - this class can be used to patch methods on live PyTorch model objects with wrappers such as `nncf.torch.dynamic_graph.context.no_nncf_trace` when doing so in the model code is not possible (e.g. if the model comes from an external library package).
+  - Compression controllers of the `nncf.api.compression.CompressionAlgorithmController` class now have a `.strip()` method that will return the compressed model object with as many custom NNCF additions removed as possible while preserving the functioning of the model object as a compressed model.
+- Fixes:
+  - Fixed statistics computation for pruned layers.
+  - (PyTorch) Fixed traced tensors to implement the YOLOv8 from Ultralytics.
+- Improvements:
+  - Extension of attributes (`transpose/permute/getitem`) for pruning node selector.
+  - NNCFNetwork was refactored from a wrapper-approach to a mixin-like approach.
+  - Added average pool 3d-like ops to pruning mask.
+  - Added Conv3d for overflow fix.
+  - `nncf.set_log_file(...)` can now be used to set location of the NNCF log file.
+  - Added support for pruning of `torch.nn.functional.pad` operation.
+  - (PyTorch) Added `torch.baddbmm` as an alias for the matmul metatype for quantization purposes.
+  - (PyTorch) Added config file for ResNet18 accuracy-aware pruning + quantization on CIFAR10.
+  - (PyTorch) Fixed JIT-traceable PyTorch models with internal patching.
+  - (PyTorch) Added `__matmul__` magic functions to the list of patched ops (for SwinTransformer by Microsoft).
+- Requirements:
+  - Updated ONNX version (1.13)
+- Docs:
+  - Multiple docs updates (fixes, improvements, etc.).
```

Comment:
It looks vague and the documentation update is a normal change. Propose to remove it